[New] Add react-chartkick@0.1.4 w/ npm auto-update

danielfernau opened the pull request:
close #12246, cc @ankane


Pull request for issue: #12246
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ankane/react-chartkick
 * Official website (optional, not the repository): https://www.chartkick.com/react
 * NPM package url (optional): https://www.npmjs.com/package/react-chartkick
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 16
   - Count of stars: 724
   - Count of forks: 24
 * NPM download stats (optional):
   - Downloads in the last day: 23
   - Downloads in the last week: 214
   - Downloads in the last month: 788

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

ankane commented:
Thanks @danielfernau 👍  One minor edit: the official website should be https://www.chartkick.com/react

danielfernau commented:
You're welcome, @ankane!
I've updated the official website link. Thanks for the hint.

sufuf3 commented on ajax/libs/react-chartkick/package.json:
```diff
@@ -0,0 +1,26 @@
+    {
+      "basePath": "dist",
+      "files": [
+        "react-chartkick*"
```
Let's modify from `"react-chartkick*"` to `"**/*"`.
We can include all the files in `dist` path.(Except there is some file we don't need, such as *.md)
Many thanks.

danielfernau commented:
@sufuf3 Thanks for your feedback!
I've squashed all my commits into a single one, updated the commit message and replaced the two .js files.

Additionally, I modified the package.json and changed `"react-chartkick*"` to `"**/*"`.

sufuf3 commented:
Helped rebase the branch to the latest master.

danielfernau commented:
@sufuf3 Thanks for your help!
